camlp4/Camlp4/Struct/Camlp4Ast2OCamlAst.ml

```diff
@@ -790,7 +794,7 @@ value varify_constructors var_names =
       | Ptyp_constr longident lst ->
           Ptyp_constr longident (List.map loop lst)
       | Ptyp_object (lst, o) ->
-          Ptyp_object (List.map (fun (s, a, t) -> (s, a, loop t)) lst, o)
+          Ptyp_object (List.map (fun (Otag s a t) -> (Otag s a (loop t))) lst, o)
```

Comment:
Code here looks like [`Ast_helper.varify_constructors`](https://github.com/ocaml/ocaml/blob/aa6b5b96a5dd41dbace23d696c2670f7a296d112/parsing/ast_helper.ml#L90). Should we match `Oinherit` here as well?